### PR TITLE
Add mdn_url, spec_url and bcd to DOMPoint.* that were missing

### PR DESCRIPTION
**Pull request by teoli2003**
The children of `DOMPoint` (`w`, `x`, `y`, `z`, `fromPoint`) were not listed. I added them.

### CODE REVIEW

File: api/DOMPoint.json [hunk 1]
```diff
@@ -98,6 +98,106 @@
           }
         }
       },
+      "fromPoint": {
+        "__compat": {
+          "description": "<code>DOMPoint()</code> constructor",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMPoint/DOMPoint/fromPoint",
```
Comment (Elchi3) [inline api/DOMPoint.json:104]: Fix URL everywhere
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMPoint/fromPoint",
```

File: api/DOMPoint.json [hunk 2]
```diff
@@ -98,6 +98,106 @@
...
+            },
+            "samsunginternet_android": {
+              "version_added": "8.0"
+            },
+            "webview_android": {
+              "version_added": "61"
+            }
+          },
+          "status": {
+            "experimental": true,
```
Comment (Elchi3) [inline api/DOMPoint.json:145]: Mark all features in this file as non-experimental.

```suggestion
            "experimental": false,
```

File: api/DOMPoint.json [hunk 3]
```diff
@@ -98,6 +98,106 @@
           }
         }
       },
+      "fromPoint": {
+        "__compat": {
+          "description": "<code>DOMPoint()</code> constructor",
```
Comment (Elchi3) [inline api/DOMPoint.json:103]: Remove everywhere
```suggestion
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]

File: api/DOMPoint.json (hunk 1 above)
Comment (teoli2003) [reply api/DOMPoint.json]: Fixed. In next commit

File: api/DOMPoint.json (hunk 2 above)
Comment (teoli2003) [reply api/DOMPoint.json]: Fixed in next commit.

File: api/DOMPoint.json (hunk 3 above)
Comment (teoli2003) [reply api/DOMPoint.json]: Fix in next commit

**Review by Elchi3 — APPROVED** [review]
Thank you! 👍

**Pull request merged**